src/battle_script_commands.c

```diff
@@ -5853,7 +5853,8 @@ static void Cmd_moveend(void)
                     if (IsBattlerAlive(battler)
                       && gBattlerAttacker != battler
                       && GetBattlerHoldEffect(battler, TRUE) == HOLD_EFFECT_EJECT_BUTTON
-                      && TARGET_TURN_DAMAGED
```

Comment:
maybe better to have a BATTLER_TURN_DAMAGED macro thats the same as TARGET_TURN_DAMAGED but for a given battler id

Reply:
Seems like it already exists `BATTLER_DAMAGED` but is only used once. Should I rename it? I like `BATTLER_TURN_DAMAGED` more

Reply:
Went ahead and made the change.

Reply:
Yeah BATTLER_DAMAGED is fine